clients/remote/remote.go

```diff
@@ -0,0 +1,97 @@
+package remote
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/influxdata/influx-cli/v2/api"
+	"github.com/influxdata/influx-cli/v2/clients"
+)
+
+type Client struct {
+	clients.CLI
+	api.RemoteConnectionsApi
+}
+
+type CreateParams struct {
+	Name             string
+	Description      string
+	OrgID            string
+	RemoteURL        string
+	RemoteAPIToken   string
+	RemoteOrgID      string
+	AllowInsecureTLS bool
+}
+
+func (c Client) Create(ctx context.Context, params *CreateParams) error {
+	// set up a struct with required params
+	body := api.RemoteConnectionCreationRequest{
+		Name:             params.Name,
+		OrgID:            params.OrgID,
+		RemoteURL:        params.RemoteURL,
+		RemoteAPIToken:   params.RemoteAPIToken,
+		RemoteOrgID:      params.RemoteOrgID,
+		AllowInsecureTLS: params.AllowInsecureTLS,
+	}
+
+	if params.Description != "" {
+		body.Description = &params.Description
+	}
+
+	if body.OrgID == "" && c.ActiveConfig.Org == "" {
+		return clients.ErrMustSpecifyOrg
+	}
+
+	if body.OrgID == "" {
+		body.OrgID = c.ActiveConfig.Org
```

Comment:
`c.ActiveConfig.Org` is a name, not an ID. You'll need to use the get-orgs API to look up the name. [Here](https://github.com/influxdata/influx-cli/blob/main/clients/bucket/create.go#L62-L76) is an example.